airflow/serialization/serialized_objects.py

```diff
@@ -1006,6 +1007,12 @@ def _serialize_node(cls, op: BaseOperator | MappedOperator, include_deps: bool)
         # Used to determine if an Operator is inherited from EmptyOperator
         serialize_op["_is_empty"] = op.inherits_from_empty_operator
 
+        if bool(op.start_trigger) ^ bool(op.next_method):
```

Comment:
```suggestion
        if not exactly_one(op.start_trigger is not None, op.next_method is not None):
```
Just for readability, no strong opinion. exactly_one can be imported here: `airflow.utils.helpers`